index.html

```diff
@@ -0,0 +1,211 @@
+            as they launch their business.
+          </p>
+        </div>
+        <div class="event-details">
+          <h3>2021.10.4(MON) ~ 6(WED)</h3>
+          <p class="venue">
+            @ Pearl Continental(PC) Hotel Bhurban, near Islamabad
+          </p>
+          <!-- <p><a href="#">Click here</a> to participate online.</p> -->
+        </div>
+      </div>
+    </section>
+    
+    <section class="second-section">
+      <div class="second-section-container">
+        <h2 class="second-section-heading">Main Program</h2>
+        <hr />
+        <div class="program-list">
+          <div class="lecture flex-box">
+            <img class="icons" src="assets/program1.png" alt="" />
+            <h2>Lecture</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!. Lorem ipsum dolor sit amet consectetur adipisicing
+              elit. A, perspiciatis!
+            </p>
+          </div>
+
+          <div class="starup-block flex-box">
+            <img class="icons" src="assets/program2.png" alt="" />
+            <h2>Building Blocks</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!. Lorem ipsum dolor sit amet consectetur adipisicing
+              elit. A, perspiciatis!.
+            </p>
+          </div>
+
+          <div class="growth-hacks flex-box">
+            <img class="icons" src="assets/program3.png" alt="" />
+            <h2>Growth Hacks</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!. Lorem ipsum dolor sit amet consectetur adipisicing
+              elit. A, perspiciatis!.
+            </p>
+          </div>
+
+          <div class="why Raise Fund flex-box">
+            <img class="icons" src="assets/program4.png" alt="" />
+            <h2>Building Blocks</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!. Lorem ipsum dolor sit amet consectetur adipisicing
+              elit. A, perspiciatis!.
+            </p>
+          </div>
+
+          <div class="forum flex-box">
+            <img class="icons" src="assets/program5.png" alt="" />
+            <h2>Forum</h2>
+            <p>
+              Lorem ipsum dolor sit amet consectetur adipisicing elit. Deserunt,
+              veritatis!. Lorem ipsum dolor sit amet consectetur adipisicing
+              elit. A, perspiciatis!.
+            </p>
+          </div>
+        </div>
+        <div class="button-container">
+          <button type="submit" class="program-button">
+            Join StartUp Lab Summit 2021
+          </button>
+        </div>
+      </div>
+    </section>
+    <div class="section-speakers-heading">
+      <h2>Featured Speakers</h2>
+      <hr />
+    </div>
```

Comment:
- [ ] In this section, you have a design problem, in the original design every single one of the speakers have their own box and the letters respect the top limit, in your design the letters exceed the top limit of the box, please fix it.

Your design:

Desktop
![imagen](https://user-images.githubusercontent.com/60631456/128282872-0248550a-042e-4796-8ecb-b8e7a243a55e.png)

Mobile
![imagen](https://user-images.githubusercontent.com/60631456/128282945-d7e97646-ec86-4d7e-8db2-c593b436ea20.png)

Original design:

Desktop
![imagen](https://user-images.githubusercontent.com/60631456/128283218-23b73a7b-7564-4b71-b102-fbd8bc1976ce.png)

Mobile
![imagen](https://user-images.githubusercontent.com/60631456/128283123-25f57bed-fb30-4aa5-a114-3638c17c9653.png)